### DISCUSSION TITLE
Data

### DISCUSSION THREAD

**Discussion by Adams-ibr** [telekom-security/tpotce#1330] [Q&A]
Please I need to know where the T-pot collects all this attack data from. Is it Real-time attack data or it's collecting it from another repo?

**Comment by e101sg** [comment]
If not mistaken, local T-pot installation  receive data from the other  T-Pot installation around the World.. Right?

**Comment by Adams-ibr** [comment]
Yeah you're right.
But I have the installation on Cloud which means is not locally installed
on either VM machine or Standalone Machine.
